spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'when creating a valid book suggestion' do
+      let(:valid_params) do
+        {
+          book_suggestion: {
+            author:    'Fake Name Lastname',
+            link:    'http://www.fakeurl.com',
+            title:    'Fake Title',
+            editorial:  'Fake Editorial',
+            price:    100,
+            publisher:  'Fake Publisher',
+            year:    2010
+          }
+        }
+      end
+      it 'creates a new suggestion' do
+        expect do
+          post :create, params: valid_params
+        end.to change { BookSuggestion.count }.by(1)
+      end
+      it 'responds with 201 status' do
+        post :create, params: valid_params
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'when creating an invalid book suggestion' do
+      let(:invalid_params) do
+        {
```

Comment:
You can use attributes_for(:book_suggestion, author: nil) for example

Reply:
Perfect!